h/presenters/group_json.py

```diff
@@ -6,9 +6,9 @@
 class GroupJSONPresenter(object):
     """Present a group in the JSON format returned by API requests."""
 
-    def __init__(self, group, links_svc=None):
-        self.group = group
-        self._links_svc = links_svc
+    def __init__(self, group_resource):
```

Comment:
Instead of having the Presenter use a service to decorate a group model object with links, the Presenter now takes a Resource, which should already have the relevant links on it.